### PR TITLE
Docker legacy

### PR DESCRIPTION
**Pull request by tlt535**
1. git clone https://github.com/htr-tech/zphisher
2.cd zphisher
3. ./zphisher.sh

### CODE REVIEW

**Comment by Ehsan9574** [comment]
apt upgrade

File: Docker/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Run ZPhiSher in Docker Compose
+
+![docker](https://img.shields.io/badge/Docker-v19.03.12-blue?style=plastic&logo=docker)
+![dockercompose](https://img.shields.io/badge/Docker_Compose-v1.25.4-orange?style=plastic&logo=docker)
```
Comment (Hackerkuttappan) [inline Docker/README.md:4]: ''' suggestion
else if line range.length>1
''''

**Review by Hackerkuttappan — CHANGES_REQUESTED** [review]

File: Docker/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Run ZPhiSher in Docker Compose
+
```
Comment (Hackerkuttappan) [inline Docker/README.md:2]: fff

**Review by Hackerkuttappan — CHANGES_REQUESTED** [review]
fff

**Comment by maryamjafar3206-byte** [comment]
Hotmail

**Comment by maryamjafar3206-byte** [comment]
Bybit

**Pull request closed without merging**